persistence-modules/spring-data-cosmosdb/src/test/java/com/baeldung/spring/data/cosmosdb/AzurecosmodbApplicationIntegrationTest.java

```diff
@@ -0,0 +1,40 @@
+package com.baeldung.spring.data.cosmosdb;
+
+import com.azure.data.cosmos.PartitionKey;
+import com.baeldung.spring.data.cosmosdb.entity.Product;
+import com.baeldung.spring.data.cosmosdb.repository.ProductRepository;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.util.Assert;
+
+import java.util.Optional;
+
+//Uncomment this when configured URI and keys for Azure Cosmos DB in application.properties
+//to run the integration test
+//@SpringBootTest
+public class AzurecosmodbApplicationIntegrationTest {
```

Comment:
name this *ManualTest and put everything ready to run without uncommenting anything (assertions included)

Reply:
I have changed the name of class to *ManualTest. I have uncommented the code in test case. I have not removed the commented part before @SpringBootTest and @Test annotation because when I was running it locally it was failing.
So if you are sure uncommenting these will not break the integration build in future.Let me know I will uncomment these.

Reply:
Hi, that's what I meant. Our pom only looks for stuff ending in IntegrationTest and others alike, so it's safe to just do it. If for some reason it fails we'll immediately know and do something about that. But in general, our readers shouldn't need to uncomment or configure stuff to get things running.

Reply:
Cool , I will uncomment these then :)

Reply:
Done